romancal/regtest/test_wfi_image_16resultants.py

```diff
@@ -0,0 +1,90 @@
+""" Roman tests for flat field correction """
+
+import pytest
+import roman_datamodels as rdm
+
+from romancal.pipeline.exposure_pipeline import ExposurePipeline
+
+# mark all tests in this module
+pytestmark = [pytest.mark.bigdata, pytest.mark.soctests]
+
+
+@pytest.fixture(scope="module")
+def run_elp(rtdata_module):
+    rtdata = rtdata_module
+
+    # The input data is from INS for stress testing at some point this should be generated
+    # every time new data is needed.
+
+    input_data = "r0000101001001001001_01101_0004_WFI01_uncal.asdf"
+    rtdata.get_data(f"WFI/image/{input_data}")
+    rtdata.input = input_data
+
+    # Test Pipeline
+    output = "r0000101001001001001_01101_0004_WFI01_cal.asdf"
+    rtdata.output = output
+    args = [
+        "roman_elp",
+        rtdata.input,
+    ]
+    ExposurePipeline.from_cmdline(args)
+    return rtdata
+
+
+@pytest.fixture(scope="module")
+def output_filename(run_elp):
+    return run_elp.output
+
+
+@pytest.fixture(scope="module")
+def output_model(output_filename):
+    with rdm.open(output_filename) as model:
+        yield model
+
+
+@pytest.fixture(scope="module")
+def input_filename(run_elp):
+    return run_elp.input
+
+
+@pytest.fixture(scope="module")
+def input_model(input_filename):
+    with rdm.open(input_filename) as model:
+        yield model
+
+
+def test_output_is_image_model(output_model):
+    # DMS280  FIXME comment says 280
+    # DMS413  FIXME log message said 413
+    # FIXME was not an assert
+    assert isinstance(output_model, rdm.datamodels.ImageModel)
```

Comment:
The previous code:
https://github.com/spacetelescope/romancal/blob/e741d1963e163d8fbbeff714cb6a2acea662ae8a/romancal/regtest/test_wfi_16resultants.py#L43-L47
Mentions DMS280 in the comment but DMS413 in the log message and was not an assert. Which requirement does this satisfy?

Reply:
413 is correct here.